CHANGELOG.md

```diff
@@ -1,8 +1,25 @@
-## 0.14.0 (unreleased)
+## 0.15.0 (unreleased)
+
+## 0.14.0 (February 03, 2020)
 
 ENHANCEMENTS:
 
 * Add REST support for teams, users, and API keys
+* various IPAM features added
+* support for rate limit "strategies" [#47](https://github.com/ns1/ns1-python/pull/47)
+
+IMPROVEMENTS:
```

Comment:
I think improvements and enhancements are the same thing . . at least according to the TF docs if that's what we're going by for these headers